README.md

```diff
@@ -109,4 +109,4 @@ of issues opened in the `ouroboros-network` repository:
 [io-classes-issues]: https://github.com/input-output-hk/ouroboros-network/issues?q=is%3Aopen+is%3Aissue+label%3Aio-classes
 [contra-tracer]: https://hackage.haskell.org/package/contra-tracer
 [io-sim-por]: https://github.com/input-output-hk/io-sim/blob/main/io-sim/how-to-use-IOSimPOR.md
-[bob-conf]: https://www.youtube.com/watch?v=e74TE0b4xEM&t=13662s
+[bob-conf]: https://youtu.be/uedUGeWN4ZM
```

Comment:
It seems already have a link to this talk here: https://github.com/input-output-hk/io-sim/pull/52/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53

Reply:
@bolt12 your link goes to this diff which set one on a recursive journey ;).